.github/config/nodejs-prod.jsonc

```diff
@@ -99,7 +99,6 @@
     "iam/deny", // PERMISSION_DENIED: Permission iam.googleapis.com/denypolicies.create denied on resource cloudresourcemanager.googleapis.com/projects/long-door-651
     "recaptcha_enterprise/snippets", // Cannot use import statement outside a module
     "run/idp-sql", // (untested) Error: Invalid contents in the credentials file
-    "run/markdown-preview/editor", // (untested) Error: could not create an identity token: Cannot fetch ID token in this environment, use GCE or set the GOOGLE_APPLICATION_CREDENTIALS environment variable to a service account credentials JSON file
```

Comment:
This isn't documented anywhere, but we've been trying to keep the -prod and -dev config files in-sync, such that tests are only run in one or the other. Could I ask you to add this exclusion to the -dev config now that these are expected to work in the prod config?